nifi-extension-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/yaml/YamlTreeReader.java

```diff
@@ -51,17 +49,12 @@ public class YamlTreeReader extends JsonTreeReader {
 
     @Override
     protected List<PropertyDescriptor> getSupportedPropertyDescriptors() {
-        final List<PropertyDescriptor> properties = new ArrayList<>(super.getSupportedPropertyDescriptors());
-        // Remove those properties which are not applicable for YAML
-        properties.remove(AbstractJsonRowRecordReader.MAX_STRING_LENGTH);
-        properties.remove(AbstractJsonRowRecordReader.ALLOW_COMMENTS);
```

Comment:
As noted in the comment, these properties do not apply to the YAML Reader. Do more recent versions of Jackson enforce string length limitations for the YAML Reader?

Reply:
During the testing I found that Yaml is using an even lower 3mb upper limit on documents:
https://github.com/FasterXML/jackson-dataformats-text/tree/2.15/yaml#maximum-input-yaml-document-size-3-mb

Reply:
Thanks for the background and reference @mark-bathori, that makes sense!